proto/sbc_provision.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+// Provision a node via the serverless backfill controller.
+package sbc_provision;
+
+option optimize_for = SPEED;
+
+message ProvisionRequest {
+    // Leave it empty for future extension.
+}
+
+message ProvisionResponse {
+    string label = 1;
+}
+
+service NodeGroupController {
```

Comment:
Shall we unify the term, such as `ResourceGroupController`? 

@shanicky PTAL

Reply:
Do we plan to provision something other than node groups? If not we can also be precise and use `NodeGroupController`.

Reply:
I'm fine with both. `Resource group` is what's used in the kernel, while `node group` is from the operator. They do have a 1-1 mapping.